stubs/aws-xray-sdk/aws_xray_sdk/core/async_context.pyi

```diff
@@ -1,15 +1,19 @@
 from _typeshed import Incomplete
+from asyncio.events import AbstractEventLoop
+from typing_extensions import ParamSpecArgs, ParamSpecKwargs
 
 from .context import Context as _Context
 
 class AsyncContext(_Context):
-    def __init__(self, *args, loop: Incomplete | None = None, use_task_factory: bool = True, **kwargs) -> None: ...
+    def __init__(
+        self, *args: ParamSpecArgs, loop: Incomplete | None = None, use_task_factory: bool = True, **kwargs: ParamSpecKwargs
+    ) -> None: ...
     def clear_trace_entities(self) -> None: ...
 
 class TaskLocalStorage:
-    def __init__(self, loop: Incomplete | None = None) -> None: ...
-    def __setattr__(self, name: str, value) -> None: ...
-    def __getattribute__(self, item: str): ...
+    def __init__(self, loop: AbstractEventLoop | None = None) -> None: ...
+    def __setattr__(self, name: str, value: str) -> None: ...
+    def __getattribute__(self, item: str) -> str: ...
```

Comment:
This can return other types than just `str`, but that's hard to express.

```suggestion
    # Forwards attribute access to task.context (if set).
    def __getattribute__(self, item: str) -> Any: ...
```